devops/azure-pipelines/discord-job-template.yml

```diff
@@ -0,0 +1,18 @@
+jobs:
```

Comment:
This task seems to run asynchronously, we may blame some sort of parallel execution that Azure is doing.

Reply:
yes, jobs are parallel by default. set `dependsOn` to modify this. you may need to use a template parameter.

also, anything here that would change in a different iteration, should be injected as a parameter and set as a variable on the pipeline. likewise verify the discord key shouldn't be secret